vite.config.js

```diff
@@ -19,7 +26,7 @@ export default defineConfig({
     manifest: true
   },
   server: {
-    port: 3001,
+    port: process.env.ELV_AI_SEARCH_PORT || 3001,
```

Comment:
you know i realize that `process.env.ELV_AI_SEARCH_PORT` is a string, and 3001 is a number, but i have been running with this locally and it works, so i guess a string is okay, heh